src/Agent.Sdk/Util/PlatformUtil.cs

```diff
@@ -171,18 +190,23 @@ private static string GetLinuxId()
                         }
                     }
                 }
+
+                throw new Exception($"Can't find ID in {filePath}");
+
             }
 
             return null;
         }
 
         private static string GetLinuxName()
         {
-            if (RunningOnLinux && File.Exists("/etc/os-release"))
+            string filePath = GetOsReleaseFilePath();
```

Comment:
should it be under the RunningOnLinux if?

Reply:
Fixed, thanks!